tests/unit/selector/isChildOfTests.js

```diff
@@ -0,0 +1,32 @@
+define([
+    'text!fixtures/swapWithChildren.html',
+    '$',
+    'src/selector/isChildOf'
+], function(fixture, $) {
+
+    var $parent = $(fixture).find('ul');
+    var $child = $parent.find('li');
+
+    describe('Is child of', function() {
+        describe('with invalid input', function() {
+            it('returns falsy when called by empty selector', function() {
+                assert.notOk($('').isChildOf('ul'));
+            });
+            it('returns falsy when passed in an empty selector', function() {
+                assert.notOk($parent.isChildOf(''));
+            });
+        });
+
+        describe('when element is not a child', function() {
+            it('returns falsy', function() {
+                assert.notOk($parent.isChildOf('ul'));
```

Comment:
Instead of `notOk`, you should check `isFalse`, as this explicitly returns `false`.